container_pipeline/model_tmp/containers.py

```diff
@@ -0,0 +1,83 @@
+import subprocess
+
+#from django.db import models
+#from django.conf import settings
+from os import path
+from json import load, dump
+
+# TODO : Change implementation into proper model and move this class into model package.
+
+
+def form_dockerfile_link(git_url, git_path, git_branch, target_file):
+    """
+    Helper function to generate docker file link.
+    :param git_url: The url of the git repository.
+    :param git_path: The path, relative to the git repository root, where the file resides.
+    :param git_branch: The repository branch where the file resides.
+    :param target_file: The name of the target file.
+    :return: The reachable link to the dockerfile.
+    """
+
+    # TODO : Move this into a lib.
+
+    link_url = None
+    if "github" in git_url or "gitlab" in git_url:
```

Comment:
do we really want to exclude all hosted git platforms except github and gitlab? cc @bamachrn

Reply:
@dharmit for now. yes. I will need to go through all the available git repos and figure out that their git path will look like. Thanks for pointing it out though i will put in an else condition

Reply:
I don't understand the reasoning about going through all available git repos. You plan to go through all available git repos for the projects that have their entries on container-index or, do you mean something else?

Reply:
No, i mean going through all the entries in the index, irrespective of the git repo they come from and having the necessary information updated in the model.

For now, we have only github and gitlab urls, but i need to go through other hosted git solutions, and figure out what their git paths will look like, so link for the same can be formed.

Reply:
@mohammedzee1000 @dharmit I agree from the feasibility point of view for now. but did we look on other SCM? I believe we can generalize this as required.